client/src/components/SearchBar/SwapLocationsButton.tsx

```diff
@@ -0,0 +1,26 @@
+import { TripQueryVariables } from '../../gql/graphql.ts';
+import swapImg from '../../static/img/swap.svg';
+
+const HINT = 'Swap from/to';
```

Comment:
I think this should be inlined, It's much clearer what it is when it's in the markup, and "HINT" is not a very helpful variable name at this location. What is it hinting?